### ISSUE TITLE
Output token stats on portal

### ISSUE THREAD

**Issue by vadaynujra** [pendulum-chain/portal#89]
### User story / Requirement
As a consumer of token analytics (like CoinGecko), I'd like to access the current state of the $PEN and $AMPE token.

### Solution option
1. There is a [script](https://github.com/pendulum-chain/bi-scripts) that extracts the following token parameters for the PEN token:

- the total amount of tokens
- the total amount of transferable tokens (tokens in circulation)
- the total locked amount of tokens
- the total reserved amount of tokens

2. Output the result of the script on a new page on website / portal. See example [here](https://apps.moonbeam.network/moonbeam/stats/).

Question: Can this output be in real time i.e. updated for each block or should I suggest a frequency of update e.g. once every 1000 blocks?

**Comment by gonzamontiel** [comment]
Hey @vadaynujra , thanks for the writeup. I believe we can rerender the value every time it changes, I don't think we need to do polling. But worst case, we can fetch it every 1 second and it'll be almost live.

**Comment by vadaynujra** [comment]
@gonzamontiel do we still need designs if we want something really simple like [this](https://apps.moonbeam.network/moonbeam/stats/)? An update every second would be better than I was hoping for but not at the cost of any performance impact, right?

**Comment by vadaynujra** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pendulum-6408a0365cda59c57d39bee5/issues/pendulum-chain/portal/89?planning-poker) with Zenhub @gonzamontiel

**Comment by vadaynujra** [comment]
@erasmus @prayagd to confirm (on return) if is required for the Portal dashboard view? If yes, new ticket to be created with relevant designs. This ticket is only about outputting the data on a page e.g. [Moonbeam token stats](https://apps.moonbeam.network/moonbeam/stats/)

**Comment by gonzamontiel** [comment]
> Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pendulum-6408a0365cda59c57d39bee5/issues/pendulum-chain/portal/89?planning-poker) with Zenhub @gonzamontiel

done!

**Comment by gonzamontiel** [comment]
> @gonzamontiel do we still need designs if we want something really simple like [this](https://apps.moonbeam.network/moonbeam/stats/)? An update every second would be better than I was hoping for but not at the cost of any performance impact, right?

I don't think we need to auto update this in the format you propose. The data would be fetched everytime you access the link.
